content/en/tests/auto_test_retries/_index.md

```diff
@@ -58,6 +54,10 @@ After you have set up Test Visibility, you can configure Auto Test Retries from
 {{< img src="continuous_integration/auto_test_retries_test_settings.png" alt="Auto Test Retries in Test Service Settings." style="width:100%" >}}
 
 The default behavior of the feature is to retry any failing test case up to 5 times.
+This behavior can be fine-tuned with the following environment variables:
+
+* `DD_CIVISIBILITY_FLAKY_RETRY_ENABLED` - set to 0 or false to explicitly disable retries even if the remote setting is enabled (default: true).
```

Comment:
We have the same vars description on line 79, right? Here might be a better place, but then we should move all of them and remove them from the other section